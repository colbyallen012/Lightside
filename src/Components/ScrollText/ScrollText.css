aside {
  background-color: pink;
  grid-area: 1 / 1 / 6 / 2;
  height: 100%;
}

.crawl-container {
  margin-top: 50px;
  font-size: 25px;
  padding: 10px;
  height: 300px;
  perspective: 150px;
}
