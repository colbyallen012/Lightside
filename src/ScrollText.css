aside {
  background-color: pink;
  grid-area: 1 / 1 / 6 / 2;
}