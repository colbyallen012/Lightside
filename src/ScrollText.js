import React, { Component } from 'react';
import './ScrollText.css';

const ScrollText = ({ movieInfo }) => {
  return (
    <aside>
      <marquee 
        behavior="scroll" 
        direction="up" 
        // style="height: 800px;"
        scrollamount="1"
      >
      { movieInfo.opening_crawl }
      </marquee>
      <h2>{ movieInfo.title }, { movieInfo.release_date }</h2>
    </aside>
  )
}

export default ScrollText;