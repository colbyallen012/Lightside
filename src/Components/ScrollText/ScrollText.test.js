import React from 'react';
import { shallow, mount } from 'enzyme';
import ScrollText from './ScrollText';
import mockMovieData from '../../Data/mockMovieData'

describe('ScrollText', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ScrollText movieInfo={mockMovieData}/>)
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});