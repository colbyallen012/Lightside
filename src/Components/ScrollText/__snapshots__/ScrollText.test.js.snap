// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollText should match snapshot 1`] = `ShallowWrapper {}`;
